app/Bookstore.Cdk/DatabaseStack.cs

```diff
@@ -38,15 +38,15 @@ internal DatabaseStack(Construct scope, string id, DatabaseStackProps props) : b
             {
                 Version = SqlServerEngineVersion.VER_14
             }),
+            StorageType = StorageType.GP3,
+            AllocatedStorage = 20,            
             Port = DatabasePort,
             SecurityGroups = new[]
             {
                 dbSG
             },
             InstanceType = InstanceType.Of(InstanceClass.BURSTABLE2, InstanceSize.MICRO),
```

Comment:
I would like to have the configuration in appSettings configurable to avoid future code change. I will take up this work if it good approach

Reply:
Added an issue for this in the issue list to discuss